### ISSUE TITLE
make prediction in unseen item GCN

### ISSUE THREAD

**Issue by qqfox** [stellargraph/stellargraph#1974]
### Description
Halo there,
I am performing the multi label classification task by using bert embedding to GCNs. Things works well except for the task of predictions. There are 2 problems:
1. Making prediction for single item from dataset proceeded with the model (from [tutorials](https://stellargraph.readthedocs.io/en/stable/demos/node-classification/gcn-node-classification.html) )
here we can make prediction for the whole dataset, however, if I tried to predict on just a single item from the dataset, it keeps raising the axis error
2. Making prediction on unseen data.
After trained and saved the model. Is it possible or any tutorial about how to make prediction on the new data (not in the dataset that used) based on the loaded model? And the data is either single items or multi items?
Many thanks for any support
### Done Checklist

- [ x] Produced code for required functionality
- [ x] Team demo

**Comment by YINTONG0090** [comment]
When I use these models to predict  new data set, there will be data dimension mismatch, I don't know how to solve it

**Comment by qqfox** [comment]
> When I use these models to predict new data set, there will be data dimension mismatch, I don't know how to solve it

me too, even when I update the feature of 1 node, it keeps predicting the label of original one. seem that the GCNs model is not able to predict on unseen data that's not in the graph (this always causes dimension issue). However, for GraphSage, I use the holdouts technique and it works. the cost is low accuracy T_T

**Comment by jonimatix** [comment]
Yes would like to know more on best approach on predicting on completely new data set? Do we need to recreate a graph network for this each time we have to predict?

**Comment by shoegazerstella** [comment]
Read [this](https://github.com/stellargraph/stellargraph/issues/1928#issuecomment-861547881) thread

**Comment by redrocket8** [comment]
> Yes would like to know more on best approach on predicting on completely new data set? Do we need to recreate a graph network for this each time we have to predict?

Do you have any update for your problem? I am also facing the same and I haven't found any solution. The models don't have a predict method for individual graphs?
